src/rules/function-url-no-scheme-relative/index.js

```diff
@@ -0,0 +1,55 @@
+import valueParser from "postcss-value-parser"
+import {
+  isStandardSyntaxValue,
+  isVariable,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "function-url-no-scheme-relative"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected scheme relative",
+})
+
+export default function (actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) { return }
+
+    root.walkDecls(function (decl) {
+      valueParser(decl.value).walk(valueNode => {
+        if (valueNode.type !== "function"
+          || valueNode.value.toLowerCase() !== "url"
+          || !valueNode.nodes.length > 0
```

Comment:
This condition would be more clear by changing the operator instead of negating it. So `valueNode.nodes.length === 0`, right?